packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-test/main/index.md

```diff
@@ -1,7 +1,16 @@
 ---
-title: Test your Okta integration
+title: Test your Okta SCIM integration
+excerpt: Test that your SCIM application can handle actual requests to Create, Read, Update and Delete (CRUD) user profile information, and run through our Okta Integration Network (OIN) quality assurance test cases.
+meta:
+  - name: description
+    content: Test that your SCIM application can handle actual requests to Create, Read, Update and Delete (CRUD) user profile information, and run through our Okta Integration Network (OIN) quality assurance test cases.
+layout: Guides
 ---
 
+This guide teaches you to how test that your SCIM application can handle actual requests to Create, Read, Update and Delete (CRUD) user profile information, and run through our Okta Integration Network (OIN) quality assurance test cases.
+
+## Scope of tests
+
```

Comment:
Does the "Prerequisites" section on the Test page also merit inclusion in the nut facts?

Reply:
I think so. Add a link to this to, from the overview nut facts.